whatsapp_archive.py

```diff
@@ -65,7 +66,12 @@ def IdentifyMessages(lines):
                 # We have a new message, so there will be no more lines for the
                 # one we've seen previously -- it's complete. Let's add it to
                 # the list.
-                messages.append((msg_date, msg_user, msg_body))
+                if "(archivo adjunto)" in msg_body or "(attached file)" in msg_body:
+                    # Determines if the current message have a file (WORK ONLY IN SPANISH AND ENGLISH)
+                    messages.append((msg_date, msg_user, msg_body, (os + "\\" + msg_body.split("(")[0][1:-1]).replace("\\", '/')))
+                else:
+                    # if not, is filled whit "0"
```

Comment:
In Python, the convention for absent data is the special type None, there are a few examples here in the code.